package.json

```diff
@@ -1,54 +1,57 @@
 {
-  "name": "mongoose-encryption",
```

Comment:
Can you keep the 2 space tabs in this file?

Reply:
Whoops, looks like prettier autoformatted it for me. Should be good now.